apps/aetx/src/aetx_env.erl

```diff
@@ -181,6 +187,18 @@ set_height(Env, X) -> Env#env{height = X}.
 
 %%------
 
+-spec set_events(env(), map()) -> env().
+set_events(Env, Events) ->
+    Env#env{events = Events}.
+
+%%------
+
+-spec accumulate_env(env(), env()) -> env().
+accumulate_env(#env{events = Events}, ToEnv) ->
```

Comment:
Badly named function, this is not accumulating anything...

Reply:
Will rename